modules/ob-installing-builds-with-operator.adoc

```diff
@@ -43,7 +43,7 @@ After installing the Builds for Red Hat OpenShift Operator, you must verify that
 
 . Click the *All instances* tab. 
 
-. On the *All instances* tab, verify that the `OpenShiftBuild` & `ShipwrightBuild` instances are listed. Alternatively, you can also click the Open Shift Build and Shipwright Build tabs.
+. On the *All instances* tab, verify that the `OpenShiftBuild` & `ShipwrightBuild` instances are listed.
```

Comment:
```suggestion
. On the *All instances* tab, verify that the `OpenShiftBuild` and `ShipwrightBuild` instances are listed.
```

We don't use `&` in general text:
- https://redhat-documentation.github.io/supplementary-style-guide/#_ampersand
- https://www.ibm.com/docs/en/ibm-style?topic=punctuation-symbols
> Do not use symbols instead of words in general text.